packages/cli/commands/project/deploy.js

```diff
@@ -94,6 +89,17 @@ exports.handler = async options => {
       );
     }
   }
+
+  try {
+    await pollDeployStatus(
+      accountId,
+      projectConfig.name,
+      deployId,
+      deployedBuildId
+    );
+  } catch (err) {
+    logger.error(err);
```

Comment:
Could you elaborate on why moving this here fixes the problem?

Also, what type of errors can exist here? Are they api-related errors, or internal JS errors? If it's the latter I think we should switch this to be `logger.debug()` so we don't fill the console with internal js errors.

Reply:
Good call on error types, I'll address that. 

Also, after looking at this again don't think think that this change is relevant here. It is in upload.js, so I'll add a comment there